src/pages/apps/apps.module.scss

```
@@ -0,0 +1,135 @@
+.title {
+  text-align: center;
+  font-size: 5rem;
+
+  @media (max-width: 768px) {
+    font-size: 3rem;
+  }
+}
+
+.subtitle {
+  text-align: center;
+  font-size: 1rem;
+}
+
+.filtersContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  position: sticky;
+  top: var(--ifm-navbar-height);
+  z-index: 100;
+  backdrop-filter: blur(10px);
+  background: rgba(255, 255, 255, 0.85);
+  padding: 1rem;
+
+  [data-theme='dark'] & {
+    background: rgba(36, 37, 38, 0.85);
+  }
+}
+
+.pillFiltersList {
+  flex-wrap: wrap;
+
+  @media (max-width: 769px) {
+    display: none;
+  }
+}
+
+.pillFilter {
+  display: flex;
+  align-items: center;
+  user-select: none;
+}
+
+.filterBadge {
+  margin-left: 0.2rem;
+  padding: 0.2rem 0.4rem;
+  font-size: 80%;
+}
```

```suggestion
.filterBadge {
  margin-left: 0.2rem;
  padding: 0.2rem 0.4rem;
  font-size: 80%;
  background-color: var(--ifm-color-emphasis-200);
  border-color: var(--ifm-color-emphasis-200);
  color: var(--ifm-color-content);
}
```

Optimising the colour scheme of Badge in dark mode

Before:

https://user-images.githubusercontent.com/8198408/186053930-3da9db47-a1a2-4c9e-b02d-4e017e853544.mp4

After:

https://user-images.githubusercontent.com/8198408/186054146-3074a568-1ba8-4046-acf9-9c1fb745a9b4.mp4